gatsby-config.js

```diff
@@ -391,6 +391,13 @@ module.exports = {
         name: "news",
       },
     },
+    {
+      resolve: "gatsby-source-filesystem",
+      options: {
+        path: `${__dirname}/src/collections/Integrations`,
```

Comment:
No need to capitalize the directory name under collections, it can be `integrations`.